.jshintignore

```diff
@@ -24,6 +24,7 @@ apps/clock/js/ext/**
 apps/clock/js/text_builder.js
 shared/js/uuid.js
 shared/js/intl/l20n*.js
+shared/js/media/jpeg-exif.js
```

Comment:
I'll file a follow up bug for that. I'd love to see it pass, but I need to do the changes upstream. Lots and lots of linting errors (like " instead of ')